### DISCUSSION TITLE
Bi-Weekly Meeting Agenda

### DISCUSSION THREAD

**Discussion by wrashi** [GothenburgBitFactory/taskwarrior#3452] [General]
With the next meeting coming up at the end of the week, we should come up with an agenda. The following proposal is just to start discussion and give us all something to push back on…

1. Welcome - 5 min - while people enter get oriented
2. Community
    - How I do it - 5 min - NEED volunteer to do a demo of how they use TaskWarrior
   - Q&A / Discussion - 5 min
3. Developers - 15 minutes
   - Update on progress over the last 2 weeks - what's been going on?
   - Questions for the community - anything you'd like to ask the (captive) audience, requests for help (i.e. beta testers, etc).
   - Roadmap - plans for the next 2 weeks or 2 years

Objections? Better ideas? Volunteers?

**Comment by djmitche** [comment]
Thanks! For the "Questions for the Community" section, I would like to highlight
 * Opportunities to be a liason for various distributions (e.g., Homebrew)
 * Trying to make a list of all of the communication channels the project has

For the roadmap, let's talk about how to manage a roadmap for a low-resources project like this. We can't really promise _anything_ will happen in 2 weeks, and most people will be working on the bits and pieces that interest them, regardless of their overall priority. That's all great and proper. But we should also have a lightweight way of having a shared understanding of what's most important at the moment, whether because it's broken or because it's preventing someone else's progress on something important.  Maybe that's GH projects, maybe it's just a conversation in this meeting, or maybe it's something else.

**Reply by ryneeverett** [reply]
> Trying to make a list of all of the communication channels the project has

A jumping off point for this would be the places where communication channels are currently documented:

- https://taskwarrior.org/support/
- https://github.com/GothenburgBitFactory/taskwarrior?tab=readme-ov-file#community

Perhaps a plan to update those will emerge.

**Reply by lauft** [reply]
I won't be able to attend the meeting at 14:00 CEST, so here some updates from my side:
* Sites `taskwarrior.org`, `timewarrior.net`, `holidata.net`, and `gothenburgbitfactory.org` have moved to GitHub pages. 
* @mattsmida  has made a POC which will help to automate the sync of the man pages between Timewarrior and timewarrior.net. ❤️ 
* My proposal for communication channels would be to reduce them to Discord and GitHub (Discussions and Issues). Regarding Discord, we may need more moderators/administrators (and know who they are). Maybe @tbabej can help out here?

**Comment by mattsmida** [comment]
Looking forward to the meeting. @wrashi, shall we use the same Google Meet link as last week?

**Reply by mattsmida** [reply]
Change of plans for me: I will not be able to make tomorrow's (May 18) meeting because of travel. I hope to share an update on the documentation cross reads at the next one.

**Reply by wrashi** [reply]
Sorry you won't be able to make it this time! For anyone else who missed the announcement of today's link, please check here…
https://github.com/GothenburgBitFactory/taskwarrior/discussions/3451

**Comment by djmitche** [comment]
By the way, I'm also happy to show how I use Taskwarrior someday. I just don't want to be the one talking all the time :)

**Reply by wrashi** [reply]
I'd love to see that demo, @djmitche!

**Comment by wrashi** [comment]
Getting the video up took longer than usual, as I've been travelling. Here it is at last!

[https://youtu.be/r5G9yiDrgkg](https://youtu.be/r5G9yiDrgkg)

**Reply by lauft** [reply]
The Discord invitation link can be found on the [Taskwarrior README](https://github.com/GothenburgBitFactory/taskwarrior/blob/
develop/README.md#community) in the Community section 👉🏻 https://discord.com/invite/eRXEHk8w62
Whatever I posted in the last meeting, this one should not expire... 🤞🏻

**Reply by wrashi** [reply]
Thanks! That's super helpful @lauft!

**Comment by wrashi** [comment]
Our next meeting is coming up in less than a week, so now seems a good time to plan the agenda. 

- @djmitche has volunteered to show how he uses TaskWarrior _if no one else wants to go first_. So, any volunteers?
- Progress from the last 2 weeks we should highlight? 
- Challenges we encountered that need discussion?
- Other topics?

**Reply by djmitche** [reply]
I've continued to mull how to plan for this project. It's become pretty clear I don't have a lot of time to work on it -- I'm just too tired of staring at a screen after work to start at this project. So it is personally a little dishonest to "plan" anything on a bi-weekly basis. But, I need some notion of what is "most important" to work on.  I think it would be useful to make that visible to others so they might direct their efforts, and I think it would be good to get multiple points of view on what is "most important".

I have mixed feelings about release-driven development, but maybe the best way to indicate this is a next-release milestone. We can be flexible about moving things into and out of the milestone, and we don't need to put a date on the release until it's ready.

So, let's plan to populate a [3.1.0 milestone](https://github.com/GothenburgBitFactory/taskwarrior/milestone/37) at this meeting.

**Reply by djmitche** [reply]
(and feel free to start early!)

**Reply by djmitche** [reply]
I also ran through _most_ of the issues and pulled out a couple of thematic topics, now [recorded as labels](https://github.com/GothenburgBitFactory/taskwarrior/labels?q=topic). I was aiming for topics that particular people are expert in -- I know next to nothing about contexts or hooks or time, for example.

**Reply by wrashi** [reply]
Saturday's meeting was nice. Dustin showed his TaskWarrior workflow - 15 years of use and improvement really shows! Lots of other great conversation, too! Check it out here…

https://youtu.be/hZQG8jDn24I

**Reply by djmitche** [reply]
Thanks as always for hosting these! I know the timeframe isn't too US-friendly, but we really would love to see anyone interested attend!
